### PR TITLE
Fix wrong category groups

### PR DESCRIPTION
**Pull request by ErebusZ**

### CODE REVIEW

**Review by PiranhaSa — APPROVED** [review]
lgtm!

File: tests/kb_test.py [hunk 1]
```diff
@@ -1026,5 +1039,23 @@ def testJsonFiles_allFilesHaveCorrectCategories_testPasses(
             categories = json_data.get("categories", {})
             standard_categories = categories.get(category, [])
 
+            assert all(group_key in CATEGORY_GROUPS for group_key in categories.keys())
```
Comment (oussamaessaji) [inline tests/kb_test.py:1042]: Why?

File: tests/kb_test.py [hunk 2]
```diff
@@ -1026,5 +1039,23 @@ def testJsonFiles_allFilesHaveCorrectCategories_testPasses(
...
             for standard_category in standard_categories:
                 assert standard_category in expected_categories
+
+
+def testJsonFiles_allFilesHaveCategories_shouldBeValid() -> None:
```
Comment (benyissa) [inline tests/kb_test.py:1047]:
```suggestion
def testJsonFiles_whenAllFilesHaveCategories_theyShouldBeValid() -> None:
```

I would make it more readable.

File: tests/kb_test.py (hunk 1 above)
Comment (mohsinenar) [inline tests/kb_test.py:1042]: assert all(...) is True

**Review by mohsinenar — DISMISSED** [review]

File: tests/kb_test.py (hunk 1 above)
Comment (ErebusZ) [reply tests/kb_test.py:1042]: whats the question?
If you are asking for the need for this then its to check that the category groups you added in a KB actually exist.

**Review by 3asm — APPROVED** [review]

**Pull request merged**